### PR TITLE
Cleanup code

### PR DESCRIPTION
**Pull request by ntindle**

### CODE REVIEW

**Comment by ntindle** [comment]
@desojo

**Comment by ntindle** [comment]
@riensen

**Comment by desojo** [comment]
copy

**Review by desojo — APPROVED** [review]

File: src/autogpt_plugins/email/email_plugin/email_plugin.py [hunk 1]
```diff
@@ -1,9 +1,9 @@
-import os
```
Comment (riensen) [inline src/autogpt_plugins/email/email_plugin/email_plugin.py:1]: Why all the reordering?

File: src/autogpt_plugins/email/__init__.py [hunk 2]
```diff
@@ -23,13 +32,6 @@ def __init__(self):
         self._description = "Auto-GPT Email Plugin: Supercharge email management."
 
     def post_prompt(self, prompt: PromptGenerator) -> PromptGenerator:
-        from .email_plugin.email_plugin import (
```
Comment (riensen) [inline src/autogpt_plugins/email/__init__.py:26]: Is this intended? I am not sure if this works and tests do not cover this (as we only have unit tests).
I would be reluctant to change this, also you also did not change it in the twitter/__init__

File: src/autogpt_plugins/email/email_plugin/email_plugin.py (hunk 1 above)
Comment (ntindle) [reply src/autogpt_plugins/email/email_plugin/email_plugin.py:1]: isort moved them

**Comment by ntindle** [comment]
obsoleted by other pr

**Pull request closed without merging**